src/map/itemdb.c

```diff
@@ -3357,6 +3377,7 @@ void itemdb_defaults(void)
 	itemdb->option_exists = itemdb_option_exists;
 	itemdb->reform_exists = itemdb_reform_exists;
 	itemdb->in_group = itemdb_in_group;
+	itemdb->search_group = itemdb_search_group;
```

Comment:
Please generate the required hook for this new function with the utilites from `/tools/HPMHookGen/`